packages/jellyfish-api-core/src/category/oracle.ts

```diff
@@ -93,6 +98,20 @@ export class Oracle {
   async listOracles (): Promise<string[]> {
     return await this.client.call('listoracles', [], 'number')
   }
+
+  /**
+   * Returns latest raw price updates from oracles.
+   *
+   * @param {OraclePriceFeed} [priceFeed]
+   * @return {Promise<OracleRawPrice[]>}
+   */
+  async listLatestRawPrices (priceFeed?: OraclePriceFeed): Promise<OracleRawPrice[]> {
+    if (priceFeed != null) {
+      return await this.client.call('listlatestrawprices', [priceFeed], 'bignumber')
+    } else {
+      return await this.client.call('listlatestrawprices', [], 'bignumber')
+    }
+  }
```

Comment:
just leave it undefined
```
async listLatestRawPrices (priceFeed?: OraclePriceFeed): Promise<OracleRawPrice[]> {
    return await this.client.call('listlatestrawprices', [priceFeed], 'bignumber')
  }
```

Reply:
Not working. If priceFeed value = empty object or null or undefined, this call will return an error.

Reply:
hmm.. then
```
const params = priceFeed !== undefined ? [priceFeed] : []
return await this.client.call('listlatestrawprices', params, 'bignumber')
```

Reply:
This is working. Okay, I adopt your suggestion.